packages/material-ui/src/Grid/Grid.test.js

```diff
@@ -90,21 +89,46 @@ describe('<Grid />', () => {
   });
 
   describe('gutter', () => {
-    it('should generate the right values', () => {
-      const defaultTheme = createMuiTheme();
+    it('should generate the right values', function test() {
+      if (/jsdom/.test(window.navigator.userAgent)) this.skip();
+
+      const remValue = 16;
       const remTheme = createMuiTheme({
         spacing: (factor) => `${0.25 * factor}rem`,
       });
 
-      expect(styles(remTheme)['spacing-xs-2']).to.deep.equal({
-        margin: '-0.25rem',
-        width: 'calc(100% + 0.5rem)',
-        '& > $item': { padding: '0.25rem' },
+      const { container, getByTestId } = render(
+        <ThemeProvider theme={remTheme}>
+          <Grid container spacing={2}>
+            <Grid item data-testid="first-custom-theme" />
+            <Grid item />
+          </Grid>
+        </ThemeProvider>,
+      );
+
+      expect(container.firstChild).toHaveComputedStyle({
+        margin: `${-1 * remValue * 0.25}px`, // '-0.25rem'
+        // width: `${750 + remValue * 0.5}px`, // 'calc(100% + 0.5rem)'
```

Comment:
@eps1lon do you maybe know what is the best way for testing style values like this? I wanted to replicate the previous test, but locally and on CI, the `100%` inside the `calc` results in different values.

Reply:
> but locally and on CI, the 100% inside the calc results in different values.

To confirm: They are different within the same task e.g `yarn test:karma` fails locally but passes in CI?

Reply:
The other way around but yes, locally it was passing with 100% = 750px, on CI was failing as 100% was 1000px I think.

Reply:
Here is the failing build https://app.circleci.com/pipelines/github/mui-org/material-ui/34076/workflows/aff0c0e1-99b5-44e0-b721-cd00826481c2/jobs/213794 locally `test:karma` was passing

Reply:
Looking into it

Reply:
Fixed in https://github.com/mnajdova/material-ui/pull/22 (ran `yarn test:karma` locally to verify).

Reply:
Work as expected, perfect! Thanks a lot!